microsite/siteConfig.js

```diff
@@ -9,7 +9,7 @@
 // site configuration options.
 
 // List of projects/orgs using your project for the users page.
-const users = [];
+//const users = [];
```

Comment:
Perhaps this should just be removed, along with the comment above, since it doesn't appear to be used anywhere.

Reply:
Could do.  The Docosaurus out of the box includes a way to say what users use your open source project. The use of this var was removed from the sample templates (a `users.js` and code in the `index.js`), but potentially long term I could see Backstage liking to say things on the microsite like, "Used by Spotify, Roadie, MultiNationalCompany X, InternetUnicorn Y, and MomAndPopStartup Z" in which case this could be returned?

But absolutely happy to fully cut it out!!

Reply:
We are using [this list](https://github.com/backstage/backstage/blob/master/ADOPTERS.md) for that purpose so I agree with @dtuite that it can be removed.